src/app/components/Post.tsx

```diff
@@ -0,0 +1,20 @@
+import { observable } from 'mobx'
```

Comment:
We use 'tsx' extension for elements which contain JSX, like `<div>`, `<span>` right inside JS. For normal model classes `ts` extension should be used.